src/auth/azureAuth.ts

```diff
@@ -1,18 +1,18 @@
+import { TokenCredential } from "@azure/core-auth";
+import { Environment } from "@azure/ms-rest-azure-env";
 import {
     AuthenticationSession,
-    Disposable as VsCodeDisposable,
     ProgressLocation,
     ProgressOptions,
     QuickPickItem,
+    Disposable as VsCodeDisposable,
     window,
 } from "vscode";
-import { AzureSessionProvider, ReadyAzureSessionProvider, Tenant, TokenInfo, isReady } from "./types";
-import { Environment } from "@azure/ms-rest-azure-env";
 import { getConfiguredAzureEnv } from "../commands/utils/config";
 import { Errorable, failed } from "../commands/utils/errorable";
-import { TokenCredential } from "@azure/core-auth";
```

Comment:
💡 Thoughts please: it almost looks like you have used some formatter for all these formatting changes? do we need all this? for the dependabot fix PR, jsut thinking with scope of keep this all contained.

Reply:
I used normal vscode to do my changes not sure what changed w.r.t formatter, will recheck on it. Thanks for catching it.